api/post.go

```diff
@@ -470,11 +473,11 @@ func getOpenGraphMetadata(c *Context, w http.ResponseWriter, r *http.Request) {
 	og := app.GetOpenGraphMetadata(url)
 
 	ogJSON, err := og.ToJSON()
+	openGraphDataCache.AddWithExpiresInSecs(props["url"], ogJSON, openGraphDataCacheExpireTime) // Cache would expire after 1 week
```

Comment:
This seems quite high. Maybe making it configurable will solve your problem. Let's default to 1 hour. 

No need to add this to the system console. Most people won't need to change this.